pkgs/development/tools/qtcreator/default.nix

```diff
@@ -51,13 +52,16 @@ mkDerivation rec {
 
   installFlags = [ "INSTALL_ROOT=$(out)" ] ++ optional withDocumentation "install_docs";
 
-  preConfigure = ''
+  preConfigure = ''                                                                                                                                                                                                                                     
+    substituteInPlace src/libs/libs.pro \                                                                                                                                                                                                               
+      --replace '$$[QT_INSTALL_QML]/QtQuick/Controls' '${qtquickcontrols}/${qtbase.qtQmlPrefix}/QtQuick/Controls'                                                                                                                                       
```

Comment:
a use of `$$[QT_INSTALL_QML]` (which doesn't exist in that form) was added to `libs.pro`, the fix is the same as for `plugins.pro` on the lines below